cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -68,6 +68,17 @@ trait CsvImportExport {
       case RowSource.AllDone =>
     }
 
+    // There might still be rows left after the latest flush is done, so initiate another flush
+    actorAsk(coordinatorActor, NodeCoordinatorActor.GetIngestionStats(dataset, version)) {
+      case DatasetCoordinatorActor.Stats(_, _, _, activeRows, _) =>
+        if (activeRows > 0) {
+          println(s"Still $activeRows left to flush in active memTable, trigger another flush....")
+          actorAsk(coordinatorActor, NodeCoordinatorActor.Flush(dataset, version), timeout) {
+            case NodeCoordinatorActor.Flushed =>
```

Comment:
Hmm yeah, that last one could definitely be logged.   And yes, need to add some other types ... MatchError is not that friendly of an error message  :)